### ISSUE TITLE
{{ crumb.link }} removed without warning

### ISSUE THREAD

**Issue by amphibian** [verbb/navigation#318]
### Describe the bug

Version 2.0.6 revamped breadcrumbs, but seemingly also removed the previously-functional `{{ crumb.link }}` variable without notice.

Replacing it in your templates with `<a href="{{ crumb.url }}">{{ crumb.title }}</a>` resolves this, but I thought I'd raise it here for anyone else running into empty breadcrumbs after updating to 2.0.6.

### Steps to reproduce

1. Update to 2.0.6
2. See empty breadcrumbs when using `{{ crumb.link }}`.

### Craft CMS version

4.2.5.1

### Plugin version

2.0.6

**Comment by engram-design** [comment]
Apologies for the breaking change. The reason for the change is that the breadcrumbs no longer include _just_ elements. Previously, the function returned an array of elements, which have a `getLink()` function.

To get the fix early, change your `verbb/navigation` requirement in `composer.json` to:

```
"require": {
  "verbb/navigation": "dev-craft-4 as 2.0.6",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.7](https://github.com/verbb/navigation/releases/tag/2.0.7)
